src/quicr_client_raw_session.cpp

```diff
@@ -492,11 +502,13 @@ ClientRawSession::publishNamedObject(const quicr::Name& quicr_name,
     }
 
     // Send last fragment, which will be less than max_transport_data_size
-    if (frag_remaining_bytes) {
+    if (frag_remaining_bytes > 0) {
       messages::MessageBuffer msg;
+      // NOLINTNEXTLINE(hicpp-signed-bitwise)
```

Comment:
Interesting... does clang-tidy shut up if the code is changed to
```cpp
datagram.header.offset_and_fin = uintVar_t((offset << 1u) + 1u);
```
?

Reply:
Yep, good call.  (It wants `U`, not `u`, but whatever.)